ui/src/app/components/prepare-migration/prepare-migration.component.ts

```diff
@@ -17,24 +18,36 @@ export class PrepareMigrationComponent implements OnInit {
   displayedColumns = ['Title', 'Source', 'Destination']
   dataSource: any = []
   migrationModes: any = []
+  subscription!: Subscription
   constructor(
     private dialog: MatDialog,
     private fetch: FetchService,
     private snack: SnackbarService,
     private targetDetailService: TargetDetailsService
-  ) {}
+  ) { }
 
   isTargetDetailSet: boolean = false
-  isStreamingCfgSet: boolean = false
   isSchemaMigration: boolean = true
   isStreamingSupported: boolean = false
+  isButtonDisabled: boolean = false
```

Comment:
Might make sense to group these flags rather than having so many independent free floating flags

Reply:
removed few flags